haproxy_agent.py

```diff
@@ -196,10 +197,15 @@ async def exapps_msg(path: str, headers: str, client_ip):
         await record_ip_failure(client_ip_str)
         return reply.set_txn_var("not_found", 1)
 
+    if not authorization_app_api:
+        # TO-DO: for proper encoding we should pass user_id here
+        user_id = ""
+        authorization_app_api = b64encode(f"{user_id}:{exapp_record.exapp_token}".encode())
```

Comment:
we can probably ignore encoding errors here `.encode(errors="ignore")`

Reply:
done